### ISSUE TITLE
[Assistive tech and device support] Page functionality isn't keyboard accessible. (11.05.1)

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#35291]
#### VFS team name
Public Websites

#### VFS product name
Outreach & Events Enhancements

#### Point of Contact/Reviewers
Brian DeConinck (@briandeconinck) - Accessibility

---

### Platform Issue
Page functionality isn't keyboard accessible.

### Issue Details
Pagination links cannot receive keyboard focus, ie. cannot be reached using the [Tab] key or operated using keyboard alone. This includes both Prev / Next and 1, 2, 3, etc.

### Link, screenshot or steps to recreate
1. Navigate to https://staging.va.gov/outreach-and-events/events/?selectedOption=past (filtering to display past events to ensure there are enough for pagination)
2. Use the [Tab] key to move keyboard focus between interactive elements. Tab through all of the events listed.
3. When the last event listed has keyboard focus, press [Tab] again. Note that focus jumps over the pagination links and straight to the first link in the footer.

<img width="350" alt="Keyboard focus on the last event in the list." src="https://user-images.githubusercontent.com/6844775/149374619-6ae6403c-3659-402a-ac1b-143a9e595b14.png"> <img width="350" alt="Keyboard focus on the first link in the footer." src="https://user-images.githubusercontent.com/6844775/149374658-f3c36534-d832-419b-a2e7-bdcb4378b04f.png">

### VA.gov Experience Standard
[Category Number 11, Issue Number 05](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/VA.gov-experience-standards.1683980311.html)

### Other References
WCAG SC 2.1.1_A

---

### Platform Recommendation
These are currently coded as <a> elements with no href attribute. Consider either using <button> instead or adding an appropriate href attribute (which will enable keyboard support).

### VFS Team Tasks to Complete

- [ ] Comment on the ticket if there are questions or concerns
- [ ] Close the ticket when the issue has been resolved or validated by your Product Owner. If a team has additional questions or needs Platform help validating the issue, please comment in the ticket.

**Comment by k80bowman** [comment]
This page uses the Design System Pagination React component. I've confirmed that our component as it is now written is not tabbable. We will take care of getting it fixed.
